styles/stylesheet.css

```diff
@@ -0,0 +1,144 @@
+body {
+  background-color: rgb(236,236,236);
+  align-items: center;
+  margin: 1rem;
+}
+
+.dogface {
+  height: 280px;
+  width: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: absolute;
+  border-color: rgb(29, 71, 152);
+}
+.header {
+  position: relative;
+  margin: 5px;
+  width: auto;
+}
+.logo {
+  position: relative;
+  margin: 5px;
+  width: 33%;
+}
+.buttons {
+  font-family: inherit;
+  font-size: 100%;
```

Comment:
I'm not a huge fan of using percents for font-size. Embrace rems or ems (tip: rems are easier to use)